Add kimai-streamcontroller-action

Thiritin opened the pull request:
### Checks
- [x] I tested my changes or release
- [x] I agree to the [terms of service](https://streamcontroller.github.io/docs/latest/plugin_dev/submitting/terms/)

https://github.com/Thiritin/kimai-streamcontroller-action

G4PLS commented:
Why is there a requirements.txt file? This is not needed and is as far as I know getting ignored when installing the Plugin

Manifest looks also not right.
`supported-app-versions` does not exist. We have `minimum-app-version` and `app-version`

`os` and `python-dependencies` are not needed in the manifest as they dont do anything in the app

`description` should be changed to `descriptions` to provide translations. If you cant translate in other languages you only add the english translation

`author` is not used in the manifest anymore

Also please provide a correct thumbnail, the resolution for that should be 1000x360 and it should be either PNG or JPEG

Thiritin commented:
> Why is there a requirements.txt file? This is not needed and is as far as I know getting ignored when installing the Plugin

https://streamcontroller.core447.com/streamcontroller/docs/latest/plugin_dev/submitting/submitting/#2-verify-requirements

The wiki indicated that it was needed. Is it not required to have this file?

Core447 commented:
@Thiritin First of all: Amazing work with your plugin!

I think you forgot to add the actual thumbnail png in the repo?
Just ping me here when you added it and also the things G4PLS mentioned.

Thiritin commented:
> A few more things that I just noticed. I know the Template or the Wiki is a little bit outdated, I hope those get updated at some point to reflect newer versions of StreamController at some point.
> ## Language
> 
> Could you maybe specify the language a little bit more? So using someting like `en_US` when you use United States english as an example
> 
> If youre not sure just use en_US, this is the "default" that most plugins use.
> ## Descriptions
> 
> You could also add `short-descriptions` those are visible on the store page directly while the `descriptions` only get shown when someone clicks on the plugin.
> 
> `descriptions` and `short-descriptions` work the exact same with the only difference that `short-descriptions` should be, well, shorter
> ## App Version
> 
> Could you provide an `app-version`? This makes it so StreamController knows the app-version that this plugin uses. You can just look at the version you developed with (Example: 1.5.0.beta.11) and paste that in there
> ## Github
> 
> I just noticed you are missing the `github` section aswell. In there you just add the link to the github repo. In your case: https://github.com/Thiritin/kimai-streamcontroller-action
> ## Manifest/About
> 
> In general If you want to get a better overview what you can add into the mainfest or the about.json file you can look at this plugin: https://github.com/G4PLS/AudioControl In there I have everything you can add to the manifest, the about.json is missing a few things.
> 
> The `about.json` file is completely optional.

I have updated the manifest and added the about.json :smile:

G4PLS commented:
Only have 2 things about the `about.json`

You provide an empty Contributors row, you can just completely remove that. Everything in `about.json` is optional.

The same with the `release-notes`, you set a path but there is no actual changelog file
If you dont want to create one your can just completely remove that section aswell

Other than that everything looks good to me

Core447 commented:
> Resolved merge conflict and approved the PR @Core447 :-)

Nice. Sorry that I didn't said this in my last message: Can you please remove the requirements.txt file? Cause if it's present SC will install the mentioned requirements into its own environment which is not ideal.

Will merge after that - sorry again
